[camera] Add ability to concurrently record and stream video (#6290)
flutter/plugins: packages/camera/camera_windows/CHANGELOG.md

## NEXT

> * Update to latest camera platform interface, and fail if user attempts to use streaming with recording (since streaming is currently unsupported on web).

Reviewer: Please use the present tense for release notes. Also `web` should be Windows here.

```suggestion
* Updates to latest camera platform interface but fails if user attempts to use streaming with recording (since streaming is currently unsupported on Windows).
```
